src/styles/annotator/annotator.scss

```diff
@@ -10,9 +10,6 @@
 // -----------------
 @use '@hypothesis/frontend-shared/styles';
 
-// Shared button styles (TEMPORARY)
```

Comment:
This was crufty (should have been removed earlier). It caused breakage when I removed the entire `shared` SASS module entrypoint because it was unneeded.